### PR TITLE
Issue #13345: Enable Empty Block Check

### PR DESCRIPTION
**Pull request by suniti0804**
Issue #13345

### CODE REVIEW

**Comment by suniti0804** [comment]
@romani please add hacktoberfest-accepted tag to this PR

File: pom.xml [hunk 1]
```diff
@@ -1329,7 +1329,7 @@
                   <version>${java.version}</version>
                 </requireJavaVersion>
                 <requireMavenVersion>
-                  <version>3.3.9</version>
+                  <version>3.5.4</version>
```
Comment (romani) [inline pom.xml:1332]: Please revert this change, if you have problems with this version please create separate issue to discuss it and if we approve issue we can do update, but let's do this in separate PR.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: pom.xml (hunk 1 above)
Comment (suniti0804) [reply pom.xml]: done

Comment (suniti0804) [reply pom.xml]: PR for maven version update - https://github.com/checkstyle/checkstyle/pull/13887

**Comment by suniti0804** [comment]
done

**Review by romani — APPROVED** [review]
Ok to merge

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/Example1.java [hunk 2]
```diff
@@ -6,17 +6,23 @@
...
 
     } catch (Exception e) {
       // ignored
     }
   }
 }
 // xdoc section -- end
+
+
+
```
Comment (nrmancuso) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/Example1.java:27]: Please remove extra empty lines from all test inputs.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/Example1.java (hunk 2 above)
Comment (nrmancuso) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/Example1.java]: Not done, two files still have extra empty lines at the end.

**Review by nrmancuso — APPROVED** [review]
